heartbeat/postfix

```diff
@@ -44,6 +44,13 @@ USAGE="Usage: $0 {start|stop|reload|monitor|validate-all|meta-data}";
 
 ##########################################################################
 
+# Check availability of the runuser command, otherwise use sudo
+if [ -x /sbin/runuser ]; then
+    SUDO=runuser
+else
+    SUDO=sudo
```

Comment:
I think we should stay with su, as that's what all the agents are using (except pingd).

So if there are issues with new su on Ubuntu you should report the issue to them, and if there's an easy fix we can use it in our agents (if it doesnt break for older versions or similar).

Reply:
Making a revision now to change it back to su.

As it turns out, if the user (postfix) doesn't exist in /etc/shadow, su will give an authentication failure every time no matter what. Sudo would work in that case anyway. But that shouldn't be an issue if you create your users correctly.

Reply:
Interresting. Yeah, that shouldnt be an issue for most people.